participants/Decentralized Docker Hub.md

```diff
@@ -0,0 +1,60 @@
+# Decentralized Docker Hub
+
+## (0) Project Name and Team Slack Handles
+
+- Project Name: Decentralized Docker Hub
+- Filecoin Slack handles: viraj
+
+## (1) Brief Project Description
+
+Using Decentralized Docker Hub, you can easily push and pull docker images from IPFS and Filecoin using command line. It is powered by Powergate. You just need to clone the repo and then run the ddocker commands (push and pull).
+
+It also has support for ENS domain names. You just need to get the IPFS hash of the docker image and set it as the content hash of your ENS domain. You can then pull the docker image using ENS domain name.
+
+## (2) Link to Project UI
+
+https://github.com/viraja1/decentralized_docker_hub
+
+## (3) What does your application/UI do?
+
+Using Decentralized Docker Hub, you can easily push and pull docker images from IPFS and filecoin using command line. It is powered by Powergate. It also has support for ENS domain names.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+We are pushing docker images and not a curated dataset
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+Decentralized Docker Hub allows users to push any Docker image to IPFS and Filecoin
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+1T - 10T
```

Comment:
Could you say a little bit more about how you're planning on sourcing this data and storing it?